concept/read_api_test.go

```diff
@@ -119,7 +122,7 @@ func TestGetConceptsByIDsNon200HTTPStatus(t *testing.T) {
 
 	ctx := tidUtils.TransactionAwareContext(context.Background(), tidUtils.NewTransactionID())
 	_, err := csAPI.GetConceptsByIDs(ctx, []string{"an-id"})
-	assert.EqualError(t, err, "concept search API returned a non-200 HTTP status code: 503")
+	assert.Error(t, err)
```

Comment:
We could define an error in read_api.go and wrap it when returning. Something like:
```go
var ErrUnexpectedResponse = errors.New("concept search API returned a non-200 HTTP status code")
//....
if resp.StatusCode != http.StatusOK {
	body, err := ioutil.ReadAll(resp.Body)
	if err != nil {
		err = fmt.Errorf("Status %d : %w", resp.Status, ErrUnexpectedResponse)
	} else {
		err = fmt.Errorf("Status %d  Response %s : %w", resp.Status, string(body), ErrUnexpectedResponse)
	}
	batchConceptsLog.WithError(err).Error("Error received from concept search API")
	return nil, err
}
```